client/src/json/posts.json

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
this will change w/ what type of schema will get from the API & DB

check w/ Herman & Clarence on what the input & output data will be later today